lib/api.js

```diff
@@ -135,8 +121,9 @@ const requestAuthorizedTokens = async () => {
       await page.type('#login-username', username);
       await page.type('#login-password', password);
       await page.click('#login-button');
-      await page.waitForSelector('#auth-accept');
-      await page.click('#auth-accept');
+      await page
+        .waitForSelector('#auth-accept, *[data-testid="auth-accept"]')
+        .then(e => e.click());
```

Comment:
wouldn't this be the same as what were already doing as dom land there is only 1 element with an id of `auth-accept` ?